crypto/asn1/a_mbstr.c

```diff
@@ -342,54 +341,8 @@ static int cpy_utf8(unsigned long value, void *arg)
     return 1;
 }
 
-/* Return 1 if the character is permitted in a PrintableString */
-static int is_printable(unsigned long value)
-{
-    int ch;
-    if (value > 0x7f)
-        return 0;
```

Comment:
I've removed the `is_numeric` function.

I'm not so sure about the need for an explicit range check.  The widest character defined is 32 bits and the `ossl_isXXX` functions will be correct for these.

The caller is doing range checking for IA5, I61 and BMP strings -- the first could be updated to disable the numeric string and printable string flags as well as the IA5 one.  I still don't think it necessary.